### ISSUE TITLE
Tenderly cannot verify contract after verifying contract with a library

### ISSUE THREAD

**Issue by pegahcarter** [Tenderly/hardhat-tenderly#142]
Had this issue on a repo I'm working on and was able to reproduce it minimally.  Minimal repo [here](https://github.com/pegahcarter/tenderly-mock-lib-verification)

Basically, if you are verifying contracts that you're deploying to a tenderly fork (in my case, Optimism), all contract verifications fail after you deploy and verify a contract that uses an external library.

Error output:
```
2023-04-26 17:36:42.747 ERROR Service =>Unexpected error occurred. 
  Error reason 400 Bad Request. 
  Error context: Could not parse request body into json
2023-04-26 17:36:42.752 ERROR Service =>Error in hardhat-tenderly: Verification failed. There was an error during the API request, please contact support with the above error.
```

**Comment by beruda** [comment]
Thanks for the report, we'll take a look!

**Comment by pegahcarter** [comment]
@beruda any luck?

**Comment by pegahcarter** [comment]
@beruda ping

**Comment by pegahcarter** [comment]
@dule-git can you help

**Comment by dule-git** [comment]
Hello, as I can see from the code, you are using `@tenderly/hardhat-tenderly@1.7.2`. This issue is known and has been fixed with version `1.7.3` and above.

I cloned your repo and tried it with version `1.7.5` and it works, so you should be good with an upgrade 🚀🪐
